src/sbin/lorax-composer

```diff
@@ -44,10 +46,10 @@ def get_parser():
     parser = argparse.ArgumentParser(description="Lorax Composer API Server",
                                      fromfile_prefix_chars="@")
 
-    parser.add_argument("--host", default="127.0.0.1", metavar="HOST",
-                        help="Host or IP to bind to (127.0.0.1)")
-    parser.add_argument("--port", default=4000, metavar="PORT",
-                        help="Port to bind to (4000)")
+    parser.add_argument("--socket", default="/var/run/weldr/api.socket", metavar="SOCKET",
```

Comment:
Is there a specific reason to still use the old `/var/run/` symlink? `/run` has been the [official definition](http://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#runRuntimeVariableData) for some time, and RHEL-7 supports it.

Reply:
No specific reason. I'll change it.